src/pages/spawn/spawnHost/spawnHostModal/SetupScriptForm.tsx

```diff
@@ -25,31 +30,37 @@ export const SetupScriptForm: React.FC<SetupScriptFormProps> = ({
   onChange,
   data,
 }) => {
-  const { setUpScript, useProjectSetupScript, spawnHostsStartedByTask } = data;
+  const {
+    setUpScript,
+    useProjectSetupScript,
+    spawnHostsStartedByTask,
+    taskSync,
+  } = data;
 
   const { search } = useLocation();
   const queryParams = parseQueryString(search);
-  const { taskId, distroId } = queryParams;
+  const { distroId } = queryParams;
+  const taskIdQueryParam = getString(queryParams.taskId);
```

Comment:
why was this renamed?

Reply:
I was getting confused between the taskId from from reducer state and the taskId from the URL params

Reply:
I wasn't able to access queryParams.taskId as a string without checking it's type. queryParams is a ParsedString https://github.com/evergreen-ci/spruce/blob/master/src/utils/queryString/parseQueryString.ts#L4 
so taskId is a `string | string[]`